README.md

```diff
@@ -97,5 +110,12 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
```

Comment:
Thanks for giving it another run through!! Did you actually run this code? It still has the typo